### DISCUSSION TITLE
Как сделать чересстрочную заливку?

### DISCUSSION THREAD

**Discussion by Davud77** [totumonline/totum-ru-issues-and-discussions#70] [Как это сделать?] [answered]
Доброго времени суток всем.

Немаловажную роль играет комформ работы с таблицами, хочется что бы все было легко читаемо для глаз, в корпоративных цветах и тд.

1. Это хотелось бы получить темное оформление интерфейса, или вообще выбирать цветовую гамму. Например так.
![image](https://user-images.githubusercontent.com/26651156/152290931-af1c7844-842c-46c0-b67a-75c71023d9b1.png)
2. Это получить такой же вид как у форматированной таблице в Excel. Где каждая 2я строка имеет полутон первой. Это очень удобно для ориентирования между строк.
![image](https://user-images.githubusercontent.com/26651156/152291398-afd04340-7a57-4e7b-a2ed-374530fb4b90.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
#### Темная тема —> в PRO

#### Чересстрочная заливка —> в MIT с 2.3.37


Форматирование таблицы:
```
f1=: setTableFormat(interlace: "5")

// or

f2=: setTableFormat(interlace: "oldlace/tomato")

// you may also use HEX instead of html-colors
```

<img width="666" alt="Снимок экрана 2022-02-03 в 20 34 59" src="https://user-images.githubusercontent.com/55755565/152398271-a9cfe3fc-b88f-40c5-889f-27c94b9b7007.png">
